src/components/Toolbar/index.js

```diff
@@ -189,7 +190,16 @@ export default class Toolbar extends Component {
   }
 
   reveal(scopeString) {
-    this.props.onReveal(scopeString);
+    swal({
+      title: 'Are you sure you want to reveal the answer?',
```

Comment:
Could we make the value of title vary based on `scopeString`. I think the current wording could be confusing to users; they might think they're revealing the entire grid.

Reply:
Ah good point! Added and also some text :)